python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -948,6 +962,47 @@ class `wcsfit.FoFClass`, associating them into matches as you go.
 
         return sourceIndices, columns
 
+    def _check_degeneracies(self, associations, extensionInfo):
```

Comment:
It looks like you could decorate it as a staticmethod as it doesn't use `self` in the body.

Reply:
It does use a config parameter actually, so I need `self`.